.claude/agents/postgres-bumper.md

```
@@ -0,0 +1,187 @@
+This gives the highest available major version (e.g. `18`).
+
+### 1c. Determine the target major
+
+- If a new major version exists upstream that is higher than the current PG_MAJOR, target the new major (major upgrade path).
+- Otherwise, stay on the current major (patch update path).
+
+### 1d. Fetch the upstream Dockerfile for the target major
+
+Determine the debian codename used by upstream for the target major:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)